language/vm/src/gas_schedule.rs

```diff
@@ -444,3 +463,25 @@ pub fn calculate_intrinsic_gas(
         min_transaction_fee.unitary_cast()
     }
 }
+
+#[allow(non_camel_case_types)]
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[repr(u8)]
+pub enum NativeCostIndex {
+    SHA2_256 = 0,
+    SHA3_256 = 1,
+    ED25519_VERIFY = 2,
+    ED25519_THRESHOLD_VERIFY = 3,
+    ADDRESS_TO_BYTES = 4,
+    U64_TO_BYTES = 5,
+    BYTEARRAY_CONCAT = 6,
+    LENGTH = 7,
+    EMPTY = 8,
+    BORROW = 9,
+    BORROW_MUT = 10,
+    PUSH_BACK = 11,
+    POP_BACK = 12,
+    DESTROY_EMPTY = 13,
+    SWAP = 14,
+    WRITE_TO_EVENT_STORE = 15,
```

Comment:
don't see save_account here, are we missing it or is there a good reason for it?